addons/mail/static/src/core/public_web/discuss_content.xml

```diff
@@ -6,21 +6,21 @@
     <div class="o-mail-DiscussContent d-flex flex-column h-100 w-100 overflow-auto o-scrollbar-thin" t-ref="root">
         <div class="o-mail-DiscussContent-header py-2 ps-2 ps-xl-3 pe-3 d-flex flex-shrink-0 align-items-center border-bottom border-secondary gap-1 z-1 flex-grow-0" t-ref="header">
             <t t-if="thread">
-                <div t-if="['channel', 'group', 'chat'].includes(thread.channel_type)" class="o-mail-DiscussContent-threadAvatar position-relative align-self-center align-items-center bg-inherit">
+                <div t-if="showThreadAvatar" class="o-mail-DiscussContent-threadAvatar position-relative align-self-center align-items-center bg-inherit">
                     <img class="rounded-2 object-fit-cover" t-att-src="thread.parent_channel_id?.avatarUrl ?? thread.avatarUrl" alt="Thread Image"/>
-                    <FileUploader t-if="!thread.parent_channel_id and thread.is_editable and thread.channel_type !== 'chat'" acceptedFileExtensions="'.bmp, .jpg, .jpeg, .png, .svg'" showUploadingText="false" multiUpload="false" onUploaded.bind="(data) => this.onFileUploaded(data)">
+                    <FileUploader t-if="isThreadAvatarEditable" acceptedFileExtensions="'.bmp, .jpg, .jpeg, .png, .svg'" showUploadingText="false" multiUpload="false" onUploaded.bind="(data) => this.onFileUploaded(data)">
                         <t t-set-slot="toggler">
                             <a href="#" class="position-absolute z-1 h-100 w-100 rounded-2 start-0 bottom-0" title="Upload Avatar">
                                 <i class="position-absolute top-50 start-50 fa fa-sm fa-pencil text-white"/>
                             </a>
                         </t>
                     </FileUploader>
-                    <ImStatus t-if="thread.channel_type === 'chat' and thread.correspondent" className="'o-mail-DiscussContent-headerImStatus position-absolute'" member="thread.correspondent"/>
+                    <ImStatus t-if="thread.correspondent and showImStatus" className="'o-mail-DiscussContent-headerImStatus position-absolute'" member="thread.correspondent"/>
                 </div>
                 <t t-else="">
                     <ThreadIcon className="'align-self-center fs-3 my-2 opacity-75'" size="'large'" thread="thread"/>
                 </t>
-                <CountryFlag t-if="thread.showCorrespondentCountry" country="thread.correspondentCountry" class="'o-mail-DiscussContent-headerCountry border shadow-sm'"/>
+                <CountryFlag t-if="showCorrespondentCountry" country="thread.correspondentCountry" class="'o-mail-DiscussContent-headerCountry border shadow-sm'"/>
```

Comment:
Making it available in `public` but not `web` doesn't make sense. Would just drop it altogether.